docs/source/user/vectorize.rst

```diff
@@ -164,8 +164,8 @@ There are two things there:
   array, a scalar (symbolically denoted by the empty tuple ``()``) and
   returns a *n*-element one-dimension array;
 
-* the list of supported concrete *signatures* as in ``@vectorize``; here we
-  only support ``int64`` arrays.
+* the list of supported concrete *signatures* are the same as in ``@vectorize``; here
```

Comment:
```suggestion
* the list of supported concrete *signatures* as per ``@vectorize``; here,
```

I think this statement needs to allude to that the provision of concrete signatures is the same as noted for `@vectorize`.